3_javascript/javascript1/week2/homework javascript1 week2/flight.js

```diff
@@ -0,0 +1,20 @@
+// Flight booking fullname function && Formal fullname //
+
+function getFullname(firstname, surname, gender, useFormalName = false) {
+  if (useFormalName === true) {
```

Comment:
A tiny nitpick: the condition of an if-statement has to be a boolean thus in this case, with `useFormalName` already being a boolean, the comparison to `true` is not needed. 😄 
```suggestion
  if (useFormalName) {
```

Reply:
changes made already